cli/cmd/run.go

```diff
@@ -0,0 +1,46 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/manifoldco/promptui"
+	"github.com/spf13/cobra"
+)
+
+var runCmd = &cobra.Command{
+	Use:   "run",
+	Short: "Run pecopeco CLI",
+	Long:  "",
+	Run: func(cmd *cobra.Command, args []string) {
+		selectOption()
+	},
+}
+
+func selectOption() {
+	prompt := promptui.Select{
+		Label: "What would you like to do?",
+		Items: []string{"search food", "show favorites", "exit"},
```

Comment:
[ホットペッパーAPI](https://webservice.recruit.co.jp/doc/hotpepper/reference.html)の大サービスエリアマスタAPI
を見た感じだと国内限定だろうから、ユーザーがよく使う部分だけでも日本語で進めるのもありかも？
今後の選択肢含めて、英語より日本語の方が日本人からすると分かりやすいし、検索結果は日本語になると思うから！

```suggestion
		Label: "何をしますか？",
		Items: []string{"飲食店を検索", "お気に入りを表示", "終了"},
```


<img width="423" alt="image" src="https://github.com/Seiya-Tagami/pecopeco/assets/92960051/19bfa33a-6c52-48c1-bbdd-085db525b3f2">

Reply:
確かに、国内限定という点では日本語でも良いかなと思いました。
@yossydev はどう思われますか？

Reply:
一旦日本語の方がいいと思いつつ、warpとかで使う場合は日本語が表示されないみたいなことなかったっけ……

- 日本語: 日本人超ありがたい。けど使うターミナルによっては日本語表示されないとかあるから逆に使い辛くなっちゃうかも？？
- 英語: 人によっては翻訳必要かもだけど、世界中にも公開できる良さがある

って感じ？？cliならエンジニアしか使わないだろうか英語でもいいかもと言う気持ちもあります！！

Reply:
あ、そっかapiが日本語返すなら諦めて日本語にするしかないか👀

Reply:
@yossydev @te19oishi 
おふたりの意見を聞いて、
CLIは基本エンジニアが使うものだと思っているので、易しい英語ならほとんどの人が読めるのではないかと思っています。使い勝手の面ではあまり問題にならないと思いました。
また、apiから返却されるデータは日本語ですが、CLI操作自体は慣習的に英語でも良いのかなと思いました。(結構好みの問題だとは思いますが)

Reply:
OKです！！👌

Reply:
ごめん返せてなかったー！！！！

> CLIは基本エンジニアが使うものだと思っているので、易しい英語ならほとんどの人が読めるのではないかと思っています。使い勝手の面ではあまり問題にならないと思いました。
> また、apiから返却されるデータは日本語ですが、CLI操作自体は慣習的に英語でも良いのかなと思いました。(結構好みの問題だとは思いますが)

いいと思いまっす！！！